app/assets/javascripts/discourse/tests/acceptance/plugin-outlet-multi-template-test.js

```diff
@@ -26,7 +24,6 @@ acceptance("Plugin Outlet - Multi Template", function (needs) {
     delete Ember.TEMPLATES[HELLO];
     // eslint-disable-next-line no-undef
     delete Ember.TEMPLATES[GOODBYE];
-    clearCache();
```

Comment:
Doesn't look like this has been introduced elsewhere... is it not needed at all?

Reply:
Also being done after each test:

https://github.com/discourse/discourse/blob/e21d2ac217e146005d8454b08f2de1e43d6dc3de/app/assets/javascripts/discourse/tests/helpers/qunit-helpers.js#L163-L163